2-Webpack/src/index.js

```diff
@@ -0,0 +1,190 @@
+    storage.set(id, title,content)
+    const itemElement = createTodoItemElement(id,title,content)
+    list.appendChild(itemElement)
+    // reset menu display
+    clearMenu()
+    setInnerText('apply-button','Add')
+}
+function deleteCheckedItems(){
+    const itemList = storage.getAll()
+    const list = document.getElementById('list')
+    for (const id in itemList) {
+        const checkbox = document.getElementById(`item-checkbox${id}`)
+        if(checkbox.checked){
+            console.log(id, 'is checked')
+            const itemElement = document.getElementById(`${id}`)
+            list.removeChild(itemElement)
+            storage.remove(id)
+        }
+    }
+}
+function cancelEdit(){
+    clearMenu()
+    editMode = false
+    setInnerText('apply-button','Add')
+    document.getElementById('cancel-button').hidden = true
+    editedID = 0  
+}
+
+function clearMenu(){
+    setValue('title-input','')
+    setValue('content-input','')
+}
+
+function menuButtonClick(){
+    if(!editMode){
+        // adding a new item
+        addItem()
+    }else{
+        // editing an existing item
+        if (getValue('title-input')==''){
+            alert('cant set an empty title')
+            return
+        }
+        const title = getValue('title-input')
+        const content = getValue('content-input')
+        setInnerText(`title${editedID}`,title)
+        setInnerText(`content${editedID}`,content)
+        storage.set(editedID, title, content)
+        clearMenu()
+        editMode = false
+        setInnerText('apply-button','Add')
+        document.getElementById('cancel-button').hidden = true
+        editedID = 0
+    }
+}
+function addButton(){
+    clearMenu()
+    document.getElementById('title-input').focus()
+}
+
+function createTodoItemElement(itemID,title,content){
+    // ol todo-item
+    const itemElement = document.createElement('ol')
+    itemElement.className = 'todo-item'
+    itemElement.id = itemID;
+    // input checkbox
+    const itemCheckboxElement = document.createElement('input')
+    itemCheckboxElement.type = 'checkbox'
+    itemCheckboxElement.className = 'item-checkbox'
+    itemCheckboxElement.id = `item-checkbox${itemID}`
+    // div todo-item-text
+    const itemTextElement = createItemTextElement(itemID,title,content)
+    const itemButtonsElement = createItemButtonsElement(itemID)
+    // append checkbox, text and buttons to list-item
+    itemElement.appendChild(itemCheckboxElement)
+    itemElement.appendChild(itemTextElement)
+    itemElement.appendChild(itemButtonsElement)
+
+    return itemElement
+}
+function createItemButtonsElement(itemID){
+
+    const itemButtonsElement = document.createElement('div')
+    itemButtonsElement.className = 'todo-item-buttons'
+    // button edit
+    const editButtonElement = document.createElement('button')
+    editButtonElement.innerText = 'edit'
+    editButtonElement.className = 'item-edit-button'
+    editButtonElement.onclick = function(){
```

Comment:
on a general level, note that `onclick` is a property of the html element, and as one it means that you can only assign **a single** event handler. consider using the `addEventListener` api if you want to make the elements more extendible